src/main/java/ch/njol/skript/classes/Parser.java

```diff
@@ -119,17 +119,7 @@ public String toCommandString(final T o) {
 	 * @return The given object's representation in a variable name.
 	 */
 	public abstract String toVariableNameString(final T o);
-	
-	/**
-	 * Returns a pattern that matches all possible outputs of {@link #toVariableNameString(Object)}. This is used to test for variable conflicts.
-	 * <p>
-	 * This pattern is inserted directly into another pattern, i.e. without any surrounding parantheses, and the pattern is compiled without any checks, thus an invalid pattern
-	 * will crash Skript.
-	 * 
-	 * @return A valid Regex pattern string matching all possible return values of {@link #toVariableNameString(Object)}
-	 */
-	public abstract String getVariableNamePattern();
```

Comment:
Don't delete it here, just deprecate it. If you remove it we're going to break ABI compatibility, and that means a lot of addons